com.hybris.hyeclipse.editor/src/com/hybris/hyeclipse/editor/copyright/page/CopyrightTabPage.java

```diff
@@ -0,0 +1,272 @@
+package com.hybris.hyeclipse.editor.copyright.page;
+
+import org.eclipse.jdt.ui.cleanup.CleanUpOptions;
+import org.eclipse.jdt.ui.cleanup.ICleanUpConfigurationUI;
+import org.eclipse.jface.dialogs.IDialogConstants;
+import org.eclipse.jface.layout.PixelConverter;
+import org.eclipse.swt.SWT;
+import org.eclipse.swt.custom.SashForm;
+import org.eclipse.swt.custom.ScrolledComposite;
+import org.eclipse.swt.events.ControlEvent;
+import org.eclipse.swt.events.ControlListener;
+import org.eclipse.swt.events.SelectionAdapter;
+import org.eclipse.swt.events.SelectionEvent;
+import org.eclipse.swt.graphics.Point;
+import org.eclipse.swt.graphics.Rectangle;
+import org.eclipse.swt.layout.GridData;
+import org.eclipse.swt.layout.GridLayout;
+import org.eclipse.swt.widgets.Button;
+import org.eclipse.swt.widgets.Composite;
+import org.eclipse.swt.widgets.Control;
+import org.eclipse.swt.widgets.Group;
+import org.eclipse.swt.widgets.Label;
+import org.eclipse.swt.widgets.Layout;
+
+import com.hybris.hyeclipse.editor.copyright.constant.CopyrightConstants;
+import com.hybris.hyeclipse.editor.copyright.manager.CopyrightManager;
+
+/**
+ * Implementation of {@link ICleanUpConfigurationUI} Save Actions tab with
+ * copyright options
+ */
+public class CopyrightTabPage implements ICleanUpConfigurationUI {
+	private static final int CLEANUP_COUNT = 2;
+	private static final double VERTICAL_SPACING_MULTIPLIER = 1.5;
+	private static final String ADD_COPYRIGHT = "Add Copyrights";
+	private static final String NO_COPYRIGHTS = "";
+	private static final String OVERRIDE_THE_COPYRIGHTS = "Override existing copyrights";
+	private PixelConverter pixelConverter;
+	private CleanUpOptions options;
+	private final CopyrightManager copyrightManager = new CopyrightManager();
+
+	public CopyrightTabPage() {
+		super();
```

Comment:
Em... you call super constructor of the interface? Why do we need this constructor anyway?